docs/source/notebooks/dap_subset.ipynb

```diff
@@ -533,10 +534,10 @@
     "# Use the `remap_label_indexers` function to convert coordinates to *positional* indexes.\n",
     "import datetime as dt\n",
     "coords = dict(lat=45, lon=290)\n",
-    "index, _ = xr.core.coordinates.remap_label_indexers(ds, coords, method=\"nearest\")\n",
+    "index = xr.core.indexing.map_index_queries(ds, coords, method=\"nearest\").dim_indexers\n",
```

Comment:
@aulemahal Just FYI, Jenkins has been failing with error below since Nov 4?  Does it means the Jupyter env needs a newer version of xarray?  A new Jupyter env is hopefully coming soon so no big deal I guess.

If you change the tutorial notebooks in the future, please keep an eye on Jenkins or better, test with Jenkins in advance.  Jenkins can test against any devel branch.  I can show you how.

```
19:58:00  _________ finch-master/docs/source/notebooks/dap_subset.ipynb::Cell 2 __________
19:58:00  Notebook cell execution failed
19:58:00  Cell 2: Cell execution caused an exception
19:58:00  
19:58:00  Input:
19:58:00  # Use the `remap_label_indexers` function to convert coordinates to *positional* indexes.
19:58:00  import datetime as dt
19:58:00  coords = dict(lat=45, lon=290)
19:58:00  index = xr.core.indexing.map_index_queries(ds, coords, method="nearest").dim_indexers
19:58:00  
19:58:00  # The `nearest` method cannot be used with slices, so we do another call for the time period.
19:58:00  ti = xr.core.indexing.map_index_queries(ds, dict(time=slice("2060-01-01", "2064-12-30"))).dim_indexers
19:58:00  
19:58:00  # Merge the spatial and temporal indices
19:58:00  index.update(ti)
19:58:00  index
19:58:00  
19:58:00  Traceback:
19:58:00  
19:58:00  ---------------------------------------------------------------------------
19:58:00  AttributeError                            Traceback (most recent call last)
19:58:00  Input In [3], in <cell line: 4>()
19:58:00        2 import datetime as dt
19:58:00        3 coords = dict(lat=45, lon=290)
19:58:00  ----> 4 index = xr.core.indexing.map_index_queries(ds, coords, method="nearest").dim_indexers
19:58:00        6 # The `nearest` method cannot be used with slices, so we do another call for the time period.
19:58:00        7 ti = xr.core.indexing.map_index_queries(ds, dict(time=slice("2060-01-01", "2064-12-30"))).dim_indexers
19:58:00  
19:58:00  AttributeError: module 'xarray.core.indexing' has no attribute 'map_index_queries'
```

Reply:
Ach sorry about that.
Yes, I updated xarray in finch and this change is not backward compatible indeed.